Sming/Libraries/ThingSpeak/ThingSpeak.cpp

```diff
@@ -0,0 +1,94 @@
+/* ThingSpeak.cpp
+ *
+ *  Created on: 01-01-2017
+ *  Author: Gustlik (Ogyb)
+ * \version 0.9
+ */
+
+#include "ThingSpeak.h"
+
+ThingSpeak::ThingSpeak()
+{
+	cleanFields();
+}
+
+ThingSpeak::ThingSpeak(String APIKey)
+{
+	writeAPIKey = APIKey;
```

Comment:
Why is the class property named writeAPIKey and not just apiKey? Are there other API keys? If not the above code can be changed to
```
this->apiKey = apiKey
```